resources/js/src/app/components/customer/ContactForm.js

```diff
@@ -67,11 +67,19 @@ Vue.component("contact-form", {
                     if (this.enableConfirmingPrivacyPolicy && !this.privacyPolicyAccepted)
                     {
                         this.privacyPolicyShowError = true;
+
+                        NotificationService.error(
+                            TranslationService.translate("Ceres::Template.contactAcceptFormPrivacyPolicy")
+                        );
+                    }
+
+                    for (const invalidField of invalidFields)
+                    {
+                        NotificationService.error(
+                            TranslationService.translate("Ceres::Template." + this._mapInvalidField(invalidField.dataset.model))
```

Comment:
Ich würde vllt nicht für jedes invalidField einen eigenen Error ausgeben. In meinem PR gibt es quasi die selbe Stelle vllt kannst du dort noch mal schauen.